index.js

```diff
@@ -0,0 +1,158 @@
+
+import { NativeModules } from 'react-native';
+import _ from 'lodash';
+
+const { RNShopify } = NativeModules;
+
+const UNKNOWN_ERROR = 'There was an unknown error. Please contact our customer support.';
+
+export default {
+  ...RNShopify,
+  getProducts: (page, collectionId, tags) => {
```

Comment:
You should have changed the order of the last two arguments, because you now have to explicitly provide a false value for `collectionId` if you just want all products with some tags. If you reversed those two arguments, then you could call this function like this:
```
getProducts(page)
getProducts(page, tags)
getProducts(page, tags, collectionId)
```
instead of:
```
getProducts(page)
getProducts(page, undefined, tags)
getProducts(page, collectionId, tags)
```
that you have now.